_includes/manuals/1.16/4.1.4_trends.md

```diff
@@ -1,12 +1,12 @@
 
 Trends in Foreman allow you to track changes in your infrastructure over time. It allows you to track both Foreman related information and any puppet facts. The Trend pages give a graph of how the number of hosts with that value have changed over time, and the current hosts list.
 
-There are two pieces to the Trends area, the Trends to track and their corresponding counters. To define trend counters, use the "Add Trend Counter" button on the '/trends' page. Optionally set the "Name" field to over-ride odd puppet fact names to be more readable. Once created you can optionally 'Edit' the Trend to change the display names of the underlying values.
+There are two pieces to the Trends area, the Trends to track and their corresponding counters. To define trend counters, you should go to Monitor->Trends, and press on "Add Trend Counter" button. Optionally set the "Name" field to over-ride odd puppet fact names to be more readable. Once created you can optionally 'Edit' the Trend to change the display names of the underlying values.
 
-Next, to start collecting trend data, set a cron job to execute 'foreman-rake trends:counter'. Each time the rake task executes it will create 1 tick on the graphs, so you can fine tune the granularity with your cron job. We recommend using the same as your puppet run interval (30 minutes). Here's an example to run once per hour:
+Next, to start collecting trend data, set a cron job to execute `foreman-rake trends:counter`. Each time the rake task executes it will create 1 tick on the graphs, so you can fine tune the granularity with your cron job. We recommend using the same as your puppet run interval (30 minutes). Here's an example to run once per hour:
```

Comment:
Generally numbers smaller than 10 are written as the word itself. So in this case changing `create 1 tick` to `create one tick`